apps/fluent-tester/src/TestComponents/Menu/MenuTest.tsx

```diff
@@ -315,6 +316,42 @@ const MenuWithCustomMenuTrigger: React.FunctionComponent = () => {
   );
 };
 
+const menuAsABlackboxStyles = StyleSheet.create({
+  actionButton: { alignSelf: 'center', justifyContent: 'center' },
+  switch: { marginHorizontal: 10 },
+});
+
+const MenuAsABlackbox: React.FunctionComponent = () => {
+  const [open, setOpen] = React.useState<boolean>(false);
+  const [dnd, setDnD] = React.useState<boolean>(false);
+
+  const onOpenChange = () => {
+    setOpen(false);
+  };
+
+  return (
+    <Stack style={stackStyle}>
+      <Menu onOpenChange={onOpenChange} open={open}>
+        <MenuTrigger>
+          <Button onClick={() => setOpen(!open)}>Blockbox Menu</Button>
+        </MenuTrigger>
+
+        <MenuPopover minWidth={200}>
+          <MenuList>
+            <Switch checked={dnd} onChange={() => setDnD(!dnd)} style={menuAsABlackboxStyles.switch} label="Don't disturb" />
```

Comment:
Please use useCallback hook for functions and don't pass them into the component directly.

Reply:
added.

Reply:
Please apply this comment on all anonymous functions passed in as props. That includes the onClick to Button and the onChanged to switch

Reply:
changed the switch change also to useCallback. 
Button uses onOpenChange that is already inside useCallback.

Reply:
I mean the Button on line 336

Reply:
my bad missed it. 
added.